src/ci/docker/host-x86_64/disabled/dist-x86_64-haiku/build-toolchain.sh

```diff
@@ -31,35 +31,35 @@ exit 1
 # First up, build a cross-compiler
 git clone --depth=1 https://git.haiku-os.org/haiku
 git clone --depth=1 https://git.haiku-os.org/buildtools
-cd $BUILDTOOLS/jam
+cd "$BUILDTOOLS/jam"
```

Comment:
Hmm, don't you also need to change lines 7 and 9 for this to do anything?

Reply:
Hi @pta2002, you marked this as resolved but didn't change the lines. If they don't need to be changed, can you explain why?